math/lrslib/Portfile

```diff
@@ -58,8 +59,14 @@ makefile.prefix_name    prefix
 
 depends_lib-append      port:gmp
 
+build.env-append        SONAME=liblrs.1.dylib \
+                        SHLIB=liblrs.${version}.dylib \
+                        SHLINK=liblrs.dylib
+
 build.target            single all-shared
 
+destroot.env            {*}${build.env}
```

Comment:
Just curious, do you want to replace any existing values for `destroot.env`? If so, no worries.

But if not, use `destroot.env-append`.